cmd/nerdctl/container_list.go

```diff
@@ -198,10 +198,14 @@ func formatAndPrintContainerInfo(containers []container.ListItem, options Format
 				return err
 			}
 		} else if options.Quiet {
-			if _, err := fmt.Fprintf(w, "%s\n", c.ID); err != nil {
+			if _, err := fmt.Fprintln(w, c.ID); err != nil {
```

Comment:
This should be in the third commit, not in the first one

Reply:
@AkihiroSuda Sorry for that, Fixed :)